workspaces/kiali/plugins/kiali-backend/CHANGELOG.md

```diff
@@ -0,0 +1,434 @@
+### Dependencies
+
+## 1.19.0
+
+### Minor Changes
+
+- d9551ae: feat(deps): update to backstage 1.31
+
+### Patch Changes
+
+- d9551ae: Change local package references to a `*`
+- d9551ae: pin the @janus-idp/cli package
+- d9551ae: upgrade to yarn v3
+- d9551ae: Change the export-dynamic script to no longer use any flags and remove the tracking of the dist-dynamic folder
+
+* **@janus-idp/cli:** upgraded to 1.15.2
+
+### Dependencies
+
+- **@janus-idp/cli:** upgraded to 1.15.1
+
+### Dependencies
+
+- **@janus-idp/cli:** upgraded to 1.15.0
+
+### Dependencies
+
+- **@janus-idp/cli:** upgraded to 1.14.0
+
+### Dependencies
+
+- **@janus-idp/cli:** upgraded to 1.13.2
+
+### Dependencies
+
+- **@janus-idp/cli:** upgraded to 1.13.1
+
+## @backstage-community/plugin-kiali-backend [1.16.0](https://github.com/janus-idp/backstage-plugins/compare/@backstage-community/plugin-kiali-backend@1.15.0...@backstage-community/plugin-kiali-backend@1.16.0) (2024-07-26)
```

Comment:
If you are going to keep this file then it should probably just match what it was from the source repo.